nautobot/docs/user-guide/core-data-model/users/savedview.md

```diff
@@ -2,7 +2,7 @@
 
 +++ 2.3.0
```

Comment:
Oh gosh, we had *three* different docs for saved views? :-) Getting rid of the user guide was a good start but IMHO saved views are not a part of the "core data model" and shouldn't exist there either. Can we get rid of this file too?

Reply:
Needed to change the `get_docs_url` function we have. I only added `savedview.md` in `core-data-model` because `get_docs_url` is expecting it there.

Reply:
```
    if hasattr(model, "documentation_static_path"):
        path = model.documentation_static_path
```

You should be able to define a `documentation_static_path` on the model and point it to the platform-functionality documentation.